mem-22-e.xml

```diff
@@ -7,7 +7,7 @@
       <column name="definition_de">der Vokal {e:sen:nolink}</column>
       <column name="definition_fa"></column>
       <column name="definition_sv">klingonskans {e:sen:nolink}-ljud</column>
-      <column name="definition_ru"></column>
+      <column name="definition_ru">буква {e}</column>
```

Comment:
It's a bit of a pain but this needs to be `{e:sen:nolink}`, the `sen` so there's no part of speech, and the `nolink` so it won't try to link or analyse the text.